com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateServiceStep.java

```diff
@@ -65,9 +65,10 @@
             .createService(client, service, StepsUtil.getSpaceId(context));
     }
 
-
     private void processServiceCreationFailure(CloudServiceExtended service, CloudOperationException e) {
         if (!service.isOptional()) {
+            getStepLogger().warn(Messages.ERROR_CREATING_SERVICE, service.getName(), service.getLabel(), service.getPlan(),
```

Comment:
Why passing more arguments than needed ?
And why not use the extended message.

Reply:
It is leftover and the last argument should not be passed.